### PR TITLE
switch to argparse for scripts

### PR DESCRIPTION
**Pull request by mmerickel**

### CODE REVIEW

**Review by stevepiercy — COMMENTED** [review]
I made an optional suggestion.

I like this improvement.

**Comment by mmerickel** [comment]
@stevepiercy what is your optional suggestion? I can't find it.

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/scripts/initialize_db.py [hunk 1]
```diff
@@ -12,19 +12,16 @@ def setup_models(dbsession):
...
 
 
-def usage(argv):
-    cmd = os.path.basename(argv[0])
-    print('usage: %s <config_uri>\n'
-          '(example: "%s development.ini")' % (cmd, cmd))
-    sys.exit(1)
+def parse_args(argv):
+    parser = argparse.ArgumentParser()
+    parser.add_argument('config_uri')
```
Comment (stevepiercy) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/scripts/initialize_db.py]:
```python
parser.add_argument('config_uri', help='Configuration file, e.g., development.ini')
```

**Review by stevepiercy — COMMENTED** [review]
this time with code

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/scripts/initialize_db.py (hunk 1 above)
Comment (mmerickel) [reply {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/scripts/initialize_db.py]: done

**Review by stevepiercy — APPROVED** [review]

**Pull request merged**